packages/http-client-csharp/emitter/src/emitter.ts

```diff
@@ -15,9 +16,15 @@ import { spawn, SpawnOptions } from "child_process";
 import fs, { statSync } from "fs";
 import { PreserveType, stringifyRefs } from "json-serialize-refs";
 import { dirname } from "path";
+import * as semver from "semver";
 import { fileURLToPath } from "url";
-import { configurationFileName, tspOutputFileName } from "./constants.js";
+import {
+  configurationFileName,
+  minVersionRequisiteForDotnet,
+  tspOutputFileName,
+} from "./constants.js";
```

Comment:
Should this be importing from the ts file?

Reply:
@JoshLove-msft  yes, TypeScript allows importing from .ts files with the correct configuration in tsconfig.json. However, we currently use .js imports for all files. I think we maintain this approach for consistency and consider changing all imports from .js to .ts together in the future.